rome-modules/src/main/java/com/rometools/modules/mediarss/io/MediaModuleParser.java

```diff
@@ -152,7 +165,12 @@ public Module parse(final Element mmRoot, final Locale locale) {
 
                     try {
                         if (content.getAttributeValue("fileSize") != null) {
-                            mc.setFileSize(Long.valueOf(content.getAttributeValue("fileSize")));
+                            String fileSizeAttr = content.getAttributeValue("fileSize").replaceAll("\\s", "");
+                            if (fileSizeAttr.matches("\\d+")) {
```

Comment:
Yup, agreed. Tried to make as little change as possible to how all the attributes are parsed inline in the method, but this makes more sense. Can't check if matcher.group(2) is null though, because that'd require a regex match, which isn't the case for pure numeric values. So did a check for \d+ as a special first case in parseFileSize.